containers/package-builder/build-virtualenv.sh.j2

```diff
@@ -8,3 +8,7 @@ mkdir --parents "$build_dir/virtualenv/"
 virtualenv --python "python{{ python_version }}" "$build_dir/virtualenv/"
 source "$build_dir/virtualenv/bin/activate"
 python -m pip install -r "$requirements_path"
+(
+    cd "$build_dir/repositories/concent/middleman_protocol/"
+    python setup.py develop
+)
```

Comment:
Same as above. Put it in a separate script so that you don't have to change the order in the `Dockerfile`.